src/discordcr/rest.cr

```diff
@@ -805,7 +805,7 @@ module Discord
     # to specify what ID to start at.
     #
     # [API docs for this method](https://discordapp.com/developers/docs/resources/guild#list-guild-members)
-    def list_guild_members(guild_id : UInt64, limit : UInt8 = 1, after : UInt64 = 0)
+    def list_guild_members(guild_id : UInt64, limit : UInt16 = 1_u16, after : UInt64 = 0_u64)
```

Comment:
@meew0 is there a reason, why you had unsigned ints in the first place here? shouldn't it be normal `Int32`s?